### PR TITLE
Changes Gangtags to Show Up As Painted

### PR DESCRIPTION
**Pull request by ArcLumin**
Fixes https://github.com/HippieStationCode/HippieStation13/issues/2879

### CODE REVIEW

File: code/game/objects/effects/decals/crayon.dm [hunk 1]
```diff
@@ -18,9 +18,10 @@
 	loc = location
 
 	name = e_name
-	desc = "A [name] drawn in crayon."
+	desc = "A [name] drawn in paint."
```
Comment (TDcoolguy300) [inline code/game/objects/effects/decals/crayon.dm]: This is effecting ALL  crayon decals, the if below renames it to be Crayon.

Comment (ArcLumin) [reply code/game/objects/effects/decals/crayon.dm]: You're right, I accidentally put it in the wrong spot, rip

**Comment by Spacedong** [comment]
"drawn in paint" wot

**Comment by ArcLumin** [comment]
Would you prefer the [name] is painted?

You can still draw things in paint though

**Comment by Spacedong** [comment]
no

**Comment by ArcLumin** [comment]
No what

I put "A painted [name]"

**Comment by Spacedong** [comment]
"A painted graffiti" makes no grammatical sense

**Comment by ArcLumin** [comment]
It doesn't say just "graffiti", if I recall

also, by that logic, the old name, "A [name] drawn in crayon" wouldn't be grammatically correct

**Comment by Kokojos** [comment]
Actually test it to make sure you got it right and this will pass right trought merging like butter in a superheated superconductor

**Comment by ArcLumin** [comment]
I'll have to wait for when I'm not phonecoding

**Comment by SplergeBucket** [comment]
yeah ok
i dont really like how the naming thing works but it's better than crayon

**Comment by doobiedoo23** [comment]
+1/3 because who the fuck sprays their tag in crayon, what is this, Baby's First Ghetto

**Comment by Kokojos** [comment]
Code's good and I approuve.

**Pull request merged**